pkg/apis/core/v1beta1/types_shoot.go

```diff
@@ -196,6 +196,17 @@ type ShootStatus struct {
 	// managed to change the hibernation settings of the cluster
 	// +optional
 	LastHibernationTriggerTime *metav1.Time `json:"lastHibernationTriggerTime,omitempty" protobuf:"bytes,16,opt,name=lastHibernationTriggerTime"`
+	// LastMaintenanceOperation holds information about the last maintenance operation on the Shoot.
+	// +optional
+	LastMaintenanceOperation *LastMaintenanceOperation `json:"lastMaintenanceOperation,omitempty" protobuf:"bytes,17,opt,name=lastMaintenanceOperation"`
+}
+
+// LastMaintenanceOperation holds information about the last maintenance operation on the Shoot.
+type LastMaintenanceOperation struct {
+	// A human readable message indicating details about the last maintenance operation.
+	Operations []string `json:"operations" protobuf:"bytes,1,rep,name=operations"`
+	// LastTriggeredTime is the most recent time when maintenance was triggered.
+	LastTriggeredTime metav1.Time `json:"lastTriggeredTime" protobuf:"bytes,2,opt,name=lastTriggeredTime"`
```

Comment:
What about making this a list, i.e. `LastMaintenances []LastMaintenance`

which contains the last 3 or 5 maintenance operations? This way, it would be quite easily trackable.